assets/pages/oneoff-contributions/helpers/ajax.js

```diff
@@ -18,6 +19,17 @@ import { checkoutError } from '../oneoffContributionsActions';
 
 const ONEOFF_CONTRIB_ENDPOINT = window.guardian.paymentApiStripeEndpoint;
 
+function stripeOneOffContributionEndpoint(testUser: ?string) {
+  if (testUser) {
+    return addQueryParamToURL(
```

Comment:
Could you use `addQueryParamsToURL`? We have a card in the backlog to delete `addQueryParamToURL` 🙂.

Reply:
Fix and changed the type of `addQueryParams` 6c63311